src/CssInliner.php

```diff
@@ -924,11 +925,16 @@ private function splitCssAndMediaQuery(string $css): array
      * Note: This method does not check whether $cssRule matches $node.
      *
      * @param \DOMElement $node
-     * @param string[][] $cssRule
+     * @param array{
+     *            media: string,
+     *            selector: string,
+     *            hasUnmatchablePseudo: bool,
+     *            declarationsBlock: string,
+     *            line: int
+     *        } $cssRule
```

Comment:
I see the original type hint indicated an array of arrays, when in fact it should have been a simple associative array.